scripts/code_style.py

```diff
@@ -125,10 +126,11 @@ def main() -> int:
     """
     Main with command line arguments.
     """
-    uncrustify_version = get_uncrustify_version()
-    if "0.75.1" not in uncrustify_version:
+    uncrustify_version = get_uncrustify_version().strip()
+    if UNCRUSTIFY_SUPPORTED_VERSION not in uncrustify_version:
         print("Warning: Using unsupported Uncrustify version '" \
-                + uncrustify_version + "'", file=STDOUT_UTF8)
+                + uncrustify_version + "' (Note: The only supported version" \
```

Comment:
Cosmetic: `\n` before `(Note` or perhaps remove the parentheses so it goes "Warning: ...\nNote: ...".